.travis.yml

```diff
@@ -46,6 +46,7 @@ jobs:
     - stage: "Additional Tests"
       env: EMBER_TRY_SCENARIO=ember-lts-2.16
     - env: EMBER_TRY_SCENARIO=ember-lts-2.18
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
```

Comment:
Can the 2.16 be removed now, or is it still in the blueprint too?

Reply:
Could be removed, it is recommended to test at least the current LTS (3.4) and the previous one (2.18), keeping 2.16 in is optional